.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
     - WP_VERSION=latest WP_MULTISITE=1
 
 install:
+    - nvm install 4 && nvm use 4
```

Comment:
There was an ESLint fatal error otherwise since Node<4 doesn't support `let`. ESLint would also warn in the Travis build about the old version of node, before ultimately it would fatal error.